inc/saibuffer.h

```diff
@@ -325,14 +325,29 @@ typedef enum _sai_buffer_profile_attr_t
      * default to 0. */
     SAI_BUFFER_PROFILE_ATTR_XOFF_TH,
 
-    /** set the buffer profile XON threshold in byte [sai_uint32_t]
+    /** set the buffer profile XON non-hysteresis threshold in byte [sai_uint32_t]
      * Valid only for ingress PG (CREATE_AND_SET)
-     * Generate XON when the total buffer usage of this PG
-     * is less this threshold and available buffer in the PG buffer
-     * is larger than the XOFF threahold.
+     * Ensures to generate XON, and ignore hysteresis, when the total 
+     * buffer usage of this PG is less than this threshold and available 
+     * buffer in the PG buffer is larger than the XOFF threshold.
+     * When the user sets both XON and XON_OFFSET thresholds, the XON
+     * trigger condition is governed by:
+     *          max(XOFF_TH - XON_OFFSET_TH, XON_TH)
```

Comment:
does the XOFF_TH here refer to SAI_BUFFER_PROFILE_ATTR_XOFF_TH? Can you clarify?